static/src/javascripts/projects/commercial/modules/article-body-adverts.ts

```diff
@@ -171,7 +188,7 @@ const addMobileInlineAds = (): Promise<boolean> => {
 				minBelow: 250,
 			},
 			' .ad-slot': adSlotClassSelectorSizes,
-			' > :not(p):not(h2):not(.ad-slot):not(#sign-in-gate)': {
+			[ignoreList]: {
```

Comment:
I don't think I'd clocked this on the first read through! I thought this was a messenger.js-like array as an object key rather than a [computed property name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#computed_property_names) 🤯 
I always thought it was funky that typescript doesn't make you put string object keys in quotes and now I'm realizing why.

Reply:
Ha, it's a bit nuts isn't it! New to me too. We use it elsewhere in the repo so I didn't feel too uncomfortable introducing it here